src/tokens-studio/spectrum2-colors/$themes.json

```diff
@@ -36,12 +36,89 @@
       "Component.table.row-hover": "7400108dc6ecce96e0b2c7801eadc2f060e77828",
       "Component.table.selected-row-background": "1149aafef0a2f857f1acf1ab8b5463fd312cc527",
       "Component.thumbnail.border": "f4ebdb86b76d6f6ce665c1893f78d586096d28e3",
-      "Icon.fill-primary": "272aca236e8bd704f5fa28d25d44024f9948ca84",
-      "Icon.background": "b3219cb28d91ab90e570dfb45e84f4cebf7e0978",
-      "Icon.blue": "8a891ab9fc2f0ef4e2d23f7d9c77cf72ef65fd12",
-      "Icon.green": "2c010cd5c6d8df15d9c05038510ba3b49b303dde",
-      "Icon.red": "1f734c4352da33631259a822a4377c36d9129856",
-      "Icon.yellow": "34ea0effbfa548c92ac7a28428625b1fc9dbc47e",
```

Comment:
@nabuhasan Why are these tokens deleted and not deprecated?

Reply:
@karstens The only token that was marked for deprecation was Icon.background, all the others were renamed to match the new structure and avoid connection issues with existing colors. I highlighted this info in the paper doc: https://paper.dropbox.com/doc/S2-Icon-colors-May-update--CPpwWR__r99dSrKDqoDtRXNlAg-BqbGyRAAL87Ehoqm9WrCB#:h2=How-we-will-modify-(potentiall

Reply:
@nabuhasan Oh, I see, I will double-check the UUIDs, I guess. 👍